src/lib.rs

```diff
@@ -0,0 +1,11 @@
+#[macro_use]
+pub mod macros;
+mod event;
+mod attribute;
+mod write;
+mod fmt;
```

Comment:
Dammit, that's what I get for trying to do a partial commit while working on something :frowning:.